src/features/dashboard/update-app/AppUpdateForm/app-update-form.scss

```diff
@@ -29,26 +29,54 @@
     display: flex;
     align-items: center;
     justify-content: center;
-
-    button {
-      margin-inline: 8px;
-    }
+    gap: rem(0.8);
   }
 
   .mblk {
```

Comment:
can you create root values and also create this style into global file to reuse this spacing on other pages/components if needed. Also use `rem` to all places as some have value in pixels and some have `rem`